packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -323,7 +323,7 @@ function createProjectionAndParams({
                     return innerHeadStr.join(" ");
                 });
                 unionStrs.push(headStrs.join(" + "));
-                unionStrs.push(") ]");
+                unionStrs.push(`) ] WHERE ${param} IS NOT NULL]`);
```

Comment:
Would it make sense to only do this if the return type is an array of non-nullable unions?

from the integration test below only for 

`search: [Search!]`

and not

`search: [Search]`

since in the latter case, it's permitted that the return array contain null (I'm not sure what cases there could be where this is desirable, but the GraphQL schema permits it).

Reply:
I can see the logic of it, but I'm not sure if this would be desired at all as `null` has no information to it. I think the expectation would be to return only non-null values from the database even if the type is not nonnull.

I believe the same reasoning is why all the return types of the generated queries and mutations are nonnull lists of nonnull types.